packages/shared/src/components/fields/Dropdown.module.css

```diff
@@ -18,3 +18,7 @@
     }
   }
 }
+
+.item:global(.h-auto) > * {
```

Comment:
To override the the context menu item, simply passing a class to it was being applied in a different element (needed one level deeper).